velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -716,7 +716,9 @@ class PlanBuilder {
   /// "row_number() over (order by b) as a"
   /// "row_number() over (partition by a order by b
   ///  rows between a + 10 preceding and 10 following)"
-  PlanBuilder& window(const std::vector<std::string>& windowFunctions);
+  PlanBuilder& window(
```

Comment:
Is it possible to pass the preSorted columns as an input to this PlanNode instead ?

It might be better to change the Window operator then to check if the pre-sorted columns is a prefix of the partition/sort by ordering it requires. In your case the pre-sorted columns matches the entire partition/sort by key so sorting can be skipped entirely. In other cases (like Presto) if its a prefix then we could do with a partial sorting approach.